src/@next/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,71 @@
+import nextId from 'react-id-generator';
+import React, { useEffect, useState } from 'react';
+import { Tab } from './Tab';
+import {
+  StyledLi,
+  StyledTabHeaderContainer,
+  StyledTabsContainer,
+  StyledUl,
+} from './TabStyle';
+
+export type TabModel = {
+  id?: string;
+  content: React.ReactNode;
+};
+
+export type TabsProps = {
+  tabs: TabModel[];
```

Comment:
can we add a description for TabModel to include what the object looks like? so it'll be easier for users to know what it expectsfrom looking at the storybook
![Screenshot 2023-01-11 at 4 09 20 PM](https://user-images.githubusercontent.com/7514754/211752189-8faa2fd8-8657-4861-be6a-a3bf33b56dcb.png)

Reply:
sure will add that

Reply:
updated